docs/docs/contracts/src/src/bridge/BitcoinTx.sol/library.BitcoinTx.md

```diff
@@ -0,0 +1,138 @@
+# BitcoinTx
```

Comment:
@nakul1010 are these files supposed to be committed? I'd imagine these to be just build files that are generated on the fly?

Reply:
yaa it shouldn't be missed it in `gitignore`